neqo-transport/src/connection/mod.rs

```diff
@@ -3545,6 +3528,27 @@ impl Connection {
     pub fn plpmtu(&self) -> usize {
         self.paths.primary().unwrap().borrow().plpmtu()
     }
+
+    fn log_packet(&self, meta: packet::MetaData, now: Instant) {
+        if log::STATIC_MAX_LEVEL == log::LevelFilter::Off || !log::log_enabled!(log::Level::Debug) {
```

Comment:
I think that the first check here is redundant:

https://docs.rs/log/0.4.25/src/log/macros.rs.html#218

```suggestion
        if !log::log_enabled!(log::Level::Debug) {
```